tasks/selinux.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+---
+- block:
+    - name: Get the ipsec tcp service ports
+      shell: |-
+        set -euo pipefail
+        firewall-cmd --info-service=ipsec | \
+          egrep " +ports: +" | sed -e "s/ *ports: //"
```

Comment:
Perhaps the regex could be a little robuster if it matches the beginning of line, e.g., `egrep "^ +ports: +"`.